docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -110,13 +107,11 @@ $ ceph orch apply -i /mnt/mon.yaml
 [NOTE]
 Applying the `mon.yaml` spec allows the existing strategy to use `labels`
 instead of `hosts`. As a result, any node with the `mon` label can host a Ceph
-mon daemon.
-Execute this step once to avoid multiple iterations when multiple Ceph Mons are
-migrated.
+Monitor daemon. Perform this step only once to avoid multiple iterations when multiple Ceph Monitors are migrated.
 
-. Check the status of the {CephCluster} and the Ceph orchestrator daemons list.
-  Make sure that the three mons are in quorum and listed by the `ceph orch`
-  command:
+. Check the status of the {CephCluster} and the Ceph Orchestrator daemons list.
+Ensure that the three Ceph Monitors are in a quorum and listed by the `ceph orch`
+command:
```

Comment:
I think this is the first time in this procedure that we're mentioning that there are 3 Ceph Monitors. Are there always supposed to be 3 Ceph Monitors? Would customers have more that they need to migrate?

Reply:
I rebased, so the change from #591 is in this PR now.